codec/codec.go

```diff
@@ -26,15 +26,15 @@ type (
 		// GetMsgAnySigners returns the signers of the given message encoded in a protobuf Any
 		// as well as the decoded google.golang.org/protobuf/proto.Message that was used to
 		// extract the signers so that this can be used in other contexts.
-		GetMsgAnySigners(msg *types.Any) ([]string, protov2.Message, error)
```

Comment:
Does changing this interface contract require a mention in the CHANGELOG?

Reply:
No, it's new anyway